configs/nuxt3.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  extends: [require.resolve('./vue3')],
+  overrides: [
+    {
+      files: [
+        '**/pages/**/*.{js,ts,vue}',
+        '**/layouts/**/*.{js,ts,vue}',
+        '**/app.{js,ts,vue}',
+        '**/error.{js,ts,vue}',
+      ],
+      rules: { 'vue/multi-word-component-names': 'off' },
```

Comment:
Waarom deze off? Misschien is het i.h.a. goed om comments bij deze overrides te zetten, met een motivatie.

Reply:
Omdat Nuxt dit wil. Nuxt staat namelijk bijv `app.vue` en `error.vue` pages toe, maar dit is in strijd met de multi-word-component-names. En daarom zet ik het voor die files uit.